web/app.js

```diff
@@ -2818,7 +2828,15 @@ function webViewerClick(evt) {
   }
 }
 
+function webViewerKeyUp(evt) {
+  if (evt.key === "Control") {
+    PDFViewerApplication._isCtrlKeyDown = false;
+  }
+}
+
 function webViewerKeyDown(evt) {
+  PDFViewerApplication._isCtrlKeyDown = evt.key === "Control";
```

Comment:
Any reason for not using (the shorter) `evt.ctrlKey` here, since that's what our existing code does?

Reply:
The idea was to be consistent with what we've in the `keyup` callback.